### ISSUE TITLE
Nvidia issue when ozone optimizations enabled

### ISSUE THREAD

**Issue by Mrestof** [kpcyrd/spotify-launcher#50]
I don't need help with this, I just want to share this info for others to not spend a lot of time on this problem.

My setup info:
I have Nvidia card and I use Wayland.

Issue Desciption:
When you launch spotify via `spotify-launcher`, it takes a few minutes to start, while taking up 100% cpu on one core during the whole period.

Reproduce steps:
1. Uncomment the following line in the config file:
  `extra_arguments = ["--enable-features=UseOzonePlatform", "--ozone-platform=wayland"]`
2. Execute the `spotify-launcher`
3. Observe the journalctl as root, you will see a lot of repeating messages like this:
  `[drm:__nv_drm_gem_nvkms_memory_prime_get_sg_table [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000200] Cannot create sg_table for NvKmsKapiMemory 0x00000000782f39e9`

Fixing steps:
1. Comment out the ozone optimization arguments

Related links:
  - [550.54.14 - Cannot create sg_table for NvKmsKapiMemory spammed when launching chrome on Wayland](https://forums.developer.nvidia.com/t/550-54-14-cannot-create-sg-table-for-nvkmskapimemory-spammed-when-launching-chrome-on-wayland/284775/25)

Hopefully this issue will pop up whenever someone searches for that log message from journalctl ;)

**Comment by kpcyrd** [comment]
Thanks for sharing! So it did work for you when running it without any extra arguments configured?

**Comment by Mrestof** [comment]
Yep, if I run it without those parameters enabled. It launches without any delay.
